### PR TITLE
Add EE, LT and LV Parsers.

### PR DESCRIPTION
**Pull request by ThierryOllivero**

### CODE REVIEW

File: feeder/feeder.py [hunk 1]
```diff
@@ -9,6 +9,9 @@
 from parsers.GB import fetch_GB
 from parsers.NO import fetch_NO
 from parsers.SE import fetch_SE
+from parsers.EE import fetch_EE
```
Comment (corradio) [inline feeder/feeder.py]: Can you order alphabetically?

File: feeder/feeder.py [hunk 2]
```diff
@@ -23,7 +26,10 @@
     fetch_FR,
     fetch_GB,
     fetch_NO,
-    fetch_SE
+    fetch_SE,
```
Comment (corradio) [inline feeder/feeder.py]: Same here :)

File: feeder/feeder.py (hunk 1 above)
Comment (ThierryOllivero) [inline feeder/feeder.py]: Done!

File: feeder/feeder.py (hunk 2 above)
Comment (ThierryOllivero) [inline feeder/feeder.py]: Done also!

**Comment by corradio** [comment]
Thanks @ThierryOllivero for the contribution! 🎉 🎈

**Pull request merged**